dashboard/src/shared/specs/mountWrapper.tsx

```diff
@@ -39,6 +38,7 @@ export const initialState = {
     kubeappsNamespace: "kubeapps",
     helmGlobalNamespace: "kubeapps-repos-global",
     carvelGlobalNamespace: "kapp-controller-packaging-global",
+    oauthLoginURI: "/oauth/login",
```

Comment:
I set this in our default state so to trigger the failures in the tests that should have caught it. Lesson: always use default settings in tests that match your prod default settings.